### ISSUE TITLE
Feat(eos_cli_config_gen): generate full ethernet_interface configuration even if in a port-channel 

### ISSUE THREAD

**Issue by ryanmerolle** [aristanetworks/ansible-avd#1847]
### Enhancement summary

Contrary to a lot of the AVD mantra, it seems guard rails were put in place to prevent ethernet_interfaces that are members of a channel group from having configuration other than the port-channel config.  

Specifically, if a port-channel is set for lacp fallback mode, no individual ethernet interface config is generated.

[LACP Fallback Documentation](https://aristanetworks.force.com/AristaCommunity/s/article/configuring-lacp-fallback-individual-ports-on-arista-switches#Comm_Kna_ka08C0000008SEUQA2_36)

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

```eos
interface Port-Channel10
        switchport trunk native vlan 300
        switchport mode trunk
        port-channel lacp fallback individual
        mlag 10
        spanning-tree portfast
interface Ethernet10
        switchport mode access
        switchport access vlan 100
        channel-group 10 mode active
        spanning-tree portfast
```

### Describe the solution you would like

If lacp port-channel lacp fallback individual is set in the parent port_channel, then allow for the ethernet_interface configuration to be generated.

### Describe alternatives you have considered

Always allow for the ethernet_interface configuration to be generated even if a member of a port-channel.

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by ryanmerolle** [comment]
This could even be considered a bug if you wanted to argue it.  I am happy to tackle this if I get some input on the approach.  Please assign this issue to me.

Jinja Template in question: [eos_cli_config_gen/templates/eos/ethernet-interfaces.j2](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2#L2-L14)

Would this have to wait for the next major release?

**Comment by ClausHolbechArista** [comment]
Is this different from what we implemented in #1335 ?

**Comment by ryanmerolle** [comment]
This issue was generated by mistake.  My coworker lead me to believe this was not in place, and I quickly raised the issue without checking.

Thanks!
